src/store/reducers/alcohol.js

```diff
@@ -1,5 +1,30 @@
-import { initialState } from '../data/alcohol';
+import * as actions from '../actions/actionTypes';
 
-const alcoholReducer = (state = initialState, action) => state;
+const setLevels = (state, action) => {
+  
+  let results = [];
+  action.drinks.map((type, key) => {
```

Comment:
are you sure you are using the map function in the right way? it looks like you were using a very old way to solve this